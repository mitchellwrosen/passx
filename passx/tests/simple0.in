[
   {
      "subject": "CSC",
      "number": "101",
      "units": 4,
      "groups": [
         {
            "lecs": [
               {
                  "section": "01",
                  "days": "M",
                  "from": 1200,
                  "to": 1250
               }
            ]
         }
      ]
   }
]
