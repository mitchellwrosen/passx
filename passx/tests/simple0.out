[{"Classes":[{"Subject":"CSC","Number":"101","Units":4,"Lec":{"Section":"01","Days":"M","From":1200,"To":1250},"Lab":null}]}]